djangocms_version_locking/monkeypatch/cms_toolbars.py

```diff
@@ -0,0 +1,36 @@
+from django.utils.translation import ugettext_lazy as _
+
+from cms.toolbar.items import ButtonList
+
+from djangocms_versioning.cms_toolbars import VersioningToolbar
+
+from djangocms_version_locking.helpers import content_is_unlocked
+
+
+_original_add_edit_button = VersioningToolbar._add_edit_button
+
+
+def _new_add_edit_button(self):
```

Comment:
Krzysztof had asked me to change my version of this monkey patch to the following:
`def new_edit_button(func):
    def inner(self, **kwargs):
        func(self, **kwargs)
        # <overrides here>
        return self
    return inner
VersioningToolbar._add_edit_button = new_edit_button(VersioningToolbar._add_edit_button)`

Reply:
I'll send you the formatted version, it's taking too long to get this to present it formatted.